src/locales/en.json

```diff
@@ -93,6 +93,19 @@
       }
     }
   },
+  "signup": {
+    "firstName": "First Name",
+    "lastName": "Last Name",
+    "email": "email",
```

Comment:
capitalization

Reply:
The capitalization is for the object value only... I did not understand what is the error?

Reply:
Revised